kubectl-plugin/test/e2e/kubectl_ray_test.go

```diff
@@ -0,0 +1,39 @@
+package e2e
+
+import (
+	"os/exec"
+	"testing"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+func TestKubectlRayGet(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Kubectl Ray Get")
+}
+
+var _ = Describe("Calling ray plugin `get` command", Ordered, func() {
+	It("succeed in getting ray cluster information", func() {
+		cmd := exec.Command("kubectl", "ray", "cluster", "get")
+		output, err := cmd.CombinedOutput()
+
+		Expect(err).NotTo(HaveOccurred())
+		Expect(output).To(ContainSubstring("raycluster-sample"))
```

Comment:
Is it possible to compare the entire output? I think it should be determinstic right?

Reply:
I believe everything but the "age" of the cluster is deterministic. I will construct the table without the age, and compare it as a substring.